src/protocol/KNXConstants.ts

```diff
@@ -52,7 +52,7 @@ export const KNX_CONSTANTS = {
 	CONNECTIONSTATE_REQUEST_TIMEOUT: 10,
 	DEVICE_CONFIGURATION_REQUEST_TIMEOUT: 10,
 	TUNNELING_REQUEST_TIMEOUT: 1,
-	CONNECTION_ALIVE_TIME: 60,
+	CONNECTION_ALIVE_TIME: 30, // CONNECTIONSTATE_REQUEST where within 60 seconds. Some Interfaces disconnects prior to 60 seconds, so we must default to 30 secs.
```

Comment:
this should be 60 according to KNX specs

Reply:
The specs indicates less than 2 minutes. If set to 60 secs, some KNX/IP gateways out there, goes in timeout and sends a DISCONNECT_REQUEST.
You'll find out that some devices don't adhere the specs 100% (even from "famous" producers, i've found one).

Reply:
Is it possible that some KNX/IP gateways you are interacting with have secure mode enabled?
The early disconnection you're experiencing could be caused by SESSION_TIMEOUT (60s) in secure mode, rather than CONNECTION_ALIVE_TIME (120s). When a secure session times out, it forces the underlying connection to close, this would explain the disconnections before 60s. 
If this were the case, the gateway expects secure session establishment and without proper secure session handling, the connection gets dropped early.